tools/fix_auth/auth_model.rb

```diff
@@ -127,9 +127,8 @@ def run(options = {})
             puts "processed #{processed} with #{errors} errors"
           end
         end
-        puts "#{options[:dry_run] ? "viewed" : "processed"} #{processed} records" unless options[:silent]
+        puts "#{options[:dry_run] ? "will change" : "changed"} #{records_changed} of #{processed} records" unless options[:silent]
```

Comment:
My brain melts with the embedded double quotes within the interpolated strings.  Can you use single quotes instead?

The `unless` is easy to miss.  I know it's how it was previously so it can be changed as a followup but maybe something like this would be easier for me to grok:

```ruby
  unless options[:silent]
    puts "#{options[:dry_run] ? 'will change' : 'changed'} #{records_changed} of #{processed} records"
  end
```

changed 5 of 10 records
will change 5 of 10 records

Maybe "would change" would be better?

Should we flip the order:

5 of 10 records changed
5 of 10 records would change (dry run enabled)